README.md

```diff
@@ -5,109 +5,184 @@ Loop end-to-end automated tests using `detox`
  - [Background reading](https://hackernoon.com/detox-gray-box-end-to-end-testing-framework-for-mobile-apps-196ccd9564ce)
 
 
-## Loop Build
-Will either build the version of loop given e.g. `build-289` or if it doesn't already exist will clone the LoopWorkspace for the given tag and then build it
+## Running the Tests Locally
 
-- `BUILD_TAG=build-289 npm run build_loop`
+### Requirements
+node ^12.5<br>
+`npm install -g detox-cli`<br>
+`npm install`<br>
 
-## Tests
+### Installing apple simulator utilities
 
-`BUILD_DIR=${PWD}/build/build-289/Build/Products CONFIG=iphone-se-2 NAME=error_1 npm run test_e2e`
+Do **not** install `applesimutils` from Homebrew. Use the custom, pre-built binary found at `bin/applesimutils`. 
```

Comment:
Maybe add a note about the need for this is to allow `critical alerts` in a system modal.